assignments/array-methods.js

```diff
@@ -54,23 +54,60 @@ const runners = [{"id":1,"first_name":"Charmain","last_name":"Seiler","email":"c
 {"id":50,"first_name":"Shell","last_name":"Baine","email":"sbaine1d@intel.com","shirt_size":"M","company_name":"Gabtype","donation":171}];
 
 // ==== Challenge 1: Use .forEach() ====
-// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName. 
+// The event director needs both the first and last names of each runner for their running bibs.  Combine both the first and last names into a new array called fullName.
 let fullName = [];
+findrunner();
 console.log(fullName);
+function findrunner(){
+  runners.forEach(function(i) {
+    fullName.push(i.first_name);
+    fullName.push(i.last_name);
+
+  });
```

Comment:
You did not have to put `.forEach` inside a function - just needed to use `.forEach` with a function to push `first_name` and `last_name` to fullName array

Reply:
For your push you can also use this syntax to combine the two. ```variable.push(param.property + " " + param.property);```